### PR TITLE
Feature/m2 1172 add success reject popups on save changes

### PR DESCRIPTION
**Pull request by alex-i-e**

### CODE REVIEW

File: src/modules/Builder/components/Popups/SaveAndPublishProcessPopup/Description/Description.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import { SavaAndPublishStep } from '../SaveAndPublishProcessPopup.types';
+import { DescriptionProps } from './Description.types';
+
+export const Description = ({ step }: DescriptionProps) => {
+  const { t } = useTranslation('app');
+  const getAppletData = useAppletData();
+  const name = getAppletData().displayName;
+
+  switch (step) {
+    case SavaAndPublishStep.AtLeast1Activity:
```
Comment (IhorNesterenko99) [inline src/modules/Builder/components/Popups/SaveAndPublishProcessPopup/Description/Description.tsx:15]: pls replace number 1 with the text "One"

File: src/modules/Builder/components/Popups/SaveAndPublishProcessPopup/SaveAndPublishProcessPopup.types.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export const enum SavaAndPublishStep {
```
Comment (IhorNesterenko99) [inline src/modules/Builder/components/Popups/SaveAndPublishProcessPopup/SaveAndPublishProcessPopup.types.ts:1]: SaveAndPublishSteps

File: src/modules/Builder/features/SaveAndPublish/SaveAndPublish.hooks.ts [hunk 3]
```diff
@@ -1,11 +1,22 @@
 import { useTranslation } from 'react-i18next';
+import { useAppDispatch } from 'redux/store';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useCallback, useEffect, useRef, useState } from 'react';
```
Comment (IhorNesterenko99) [inline src/modules/Builder/features/SaveAndPublish/SaveAndPublish.hooks.ts:4]: import from 'react' in the very first place, add correct spacings

File: src/modules/Builder/features/SaveAndPublish/SaveAndPublish.hooks.ts [hunk 4]
```diff
@@ -37,23 +49,20 @@ export const useAppletData = () => {
           [language]: appletInfo.about,
         },
         themeId: null, // TODO: create real themeId
+        // eslint-disable-next-line
+        // @ts-ignore
```
Comment (IhorNesterenko99) [inline src/modules/Builder/features/SaveAndPublish/SaveAndPublish.hooks.ts:53]: pls remove @ts-ignore

**Review by muhurov — COMMENTED** [review]
Please change all 'Sava...' names

**Comment by muhurov** [comment]
image
please check

**Review by muhurov — APPROVED** [review]

**Pull request merged**